osu.Game/Scoring/Legacy/ScoreInfoExtensions.cs

```diff
@@ -68,6 +68,22 @@ private static long convertStandardisedToClassic(int rulesetId, long standardise
             }
         }
 
+        public static double GetOsuClassicScoreMultiplier(this ScoreProcessor scoreProcessor)
```

Comment:
this is horrific

- should not be an extension method on `ScoreProcessor` at the very least, just make this plain static
- ideally would be constrained in the ruleset project rather than spill into the generic assembly (it was borderline fine until now but this sort of thing being done crosses the line for me)